eiffel-vocabulary/EiffelConfigurationChangedEvent.md

```diff
@@ -0,0 +1,54 @@
+# EiffelConfigurationChangedEvent
+The EiffelConfigurationChangedEvent signifies that the configuration of an item, typically an actor within the continuous integration and delivery pipeline, has changed.
+
+Similarly to [EiffelDocumentationCreatedEvent](EiffelDocumentationCreatedEvent.md), this event is intended for situations where configuration is not stored and treated as any other source (in which case [EiffelSourceChangeCreatedEvent](EiffelSourceChangeCreatedEvent.md) and [EiffelSourceChangeSubmittedEvent](EiffelSourceChangeSubmittedEvent.md) SHOULD be used instead of EiffelConfigurationChangedEvent).
```

Comment:
The point is that assuming a configuration as code mindset, you should be able to use SourceChangeCreated and SourceChangeSubmitted to represent configuration changes as well. In that case, the ConfigurationChanged is restricted to whenever that's not the case.